packages/circuit-ui/components/CalendarTag/CalendarTag.tsx

```diff
@@ -13,51 +13,74 @@
  * limitations under the License.
  */
 
-import { Component } from 'react';
-import PropTypes from 'prop-types';
+import { Component, createRef } from 'react';
 import styled from '@emotion/styled';
-import { START_DATE } from 'react-dates/constants';
```

Comment:
`react-dates/constants` is missing types, so I removed the import and copied the constant into the file instead.

Reply:
Exactly my thoughts.